app/guid-file/route.ts

```diff
@@ -40,11 +44,15 @@ export default class GuidFile extends Route {
         const dateModified = model.dateModified;
         const institutions = await model.target.get('affiliatedInstitutions');
         const metaTagsData = {
-            title: model.name,
+            title: this.metadata.title ? this.metadata.title : model.name,
             identifier: model.guid,
             publishedDate: dateCreated ? moment(dateCreated).format('YYYY-MM-DD') : undefined,
             modifiedDate: dateModified ? moment(dateModified).format('YYYY-MM-DD') : undefined,
             institution: institutions.map((institution: Institution) => institution.get('name')),
+            description: this.metadata.description ?
+                this.metadata.description :
+                this.intl.t('general.hosted_on_the_osf'),
```

Comment:
Not all the files we view on the guid-file route are hosted on the OSF, so is this default description valid?

Reply:
I mean… kinda? This is the default description that the meta tags used already, I'm just overriding it in the case of a description in the metadata. I suppose we could change it to "Hosted by the OSF" instead? If I change that particular translation string, then that would be the default for all of the descriptions. Thoughts?